Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test01
Start line: 12
End line: 24
Method source code: 
    public void test01() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test01");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "", "", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #6
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #8
31: ldc           #8
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test02
Start line: 27
End line: 38
Method source code: 
    public void test02() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test02");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "hi!", "hi!");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #13
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #14
29: ldc           #14
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test03
Start line: 41
End line: 52
Method source code: 
    public void test03() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test03");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "hi!", "");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #16
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #14
29: ldc           #8
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test04
Start line: 55
End line: 67
Method source code: 
    public void test04() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test04");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "hi!", "hi!", parser3);
            org.junit.Assert.fail("Expected exception of type java.nio.charset.IllegalCharsetNameException; message: hi!");
        } catch (java.nio.charset.IllegalCharsetNameException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #18
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #14
31: ldc           #14
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #20
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/nio/charset/IllegalCharsetNameException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test05
Start line: 70
End line: 82
Method source code: 
    public void test05() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test05");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "hi!", "", parser3);
            org.junit.Assert.fail("Expected exception of type java.nio.charset.IllegalCharsetNameException; message: hi!");
        } catch (java.nio.charset.IllegalCharsetNameException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #22
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #14
31: ldc           #8
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #20
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/nio/charset/IllegalCharsetNameException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test06
Start line: 85
End line: 96
Method source code: 
    public void test06() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test06");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "", "UTF-8");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #23
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #8
29: ldc           #24
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test07
Start line: 99
End line: 111
Method source code: 
    public void test07() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test07");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "UTF-8", "", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #25
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #24
31: ldc           #8
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test08
Start line: 114
End line: 125
Method source code: 
    public void test08() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test08");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            java.nio.ByteBuffer byteBuffer1 = org.jsoup.helper.DataUtil.readToByteBuffer(inputStream0);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #26
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: invokestatic  #27
30: astore_2
31: ldc           #10
33: invokestatic  #11
36: goto          40
39: astore_2
40: return
Exception table:
from    to  target type
26    36    39   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test09
Start line: 128
End line: 139
Method source code: 
    public void test09() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test09");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "", "");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #28
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #8
29: ldc           #8
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test10
Start line: 142
End line: 153
Method source code: 
    public void test10() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test10");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "", "hi!");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #29
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #8
29: ldc           #14
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test11
Start line: 156
End line: 167
Method source code: 
    public void test11() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test11");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "UTF-8", "hi!");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #30
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #24
29: ldc           #14
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test12
Start line: 170
End line: 182
Method source code: 
    public void test12() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test12");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "", "hi!", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #31
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #8
31: ldc           #14
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test13
Start line: 185
End line: 197
Method source code: 
    public void test13() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test13");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "hi!", "UTF-8", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #32
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #14
31: ldc           #24
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test14
Start line: 200
End line: 205
Method source code: 
    public void test14() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test14");
        java.lang.String str1 = org.jsoup.helper.DataUtil.getCharsetFromContentType("hi!");
        org.junit.Assert.assertNull(str1);
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #33
19: aastore
20: invokevirtual #7
23: pop
24: ldc           #14
26: invokestatic  #34
29: astore_1
30: aload_1
31: invokestatic  #35
34: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test15
Start line: 208
End line: 219
Method source code: 
    public void test15() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test15");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "", "hi!");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #36
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #8
29: ldc           #14
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test16
Start line: 222
End line: 227
Method source code: 
    public void test16() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test16");
        java.lang.String str1 = org.jsoup.helper.DataUtil.getCharsetFromContentType("UTF-8");
        org.junit.Assert.assertNull(str1);
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #37
19: aastore
20: invokevirtual #7
23: pop
24: ldc           #24
26: invokestatic  #34
29: astore_1
30: aload_1
31: invokestatic  #35
34: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test17
Start line: 230
End line: 242
Method source code: 
    public void test17() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test17");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "hi!", "hi!", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #38
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #14
31: ldc           #14
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test18
Start line: 245
End line: 256
Method source code: 
    public void test18() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test18");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "", "UTF-8");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #39
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #8
29: ldc           #24
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test19
Start line: 259
End line: 270
Method source code: 
    public void test19() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test19");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "UTF-8", "UTF-8");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #40
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #24
29: ldc           #24
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test20
Start line: 273
End line: 284
Method source code: 
    public void test20() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test20");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "hi!", "");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #41
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #14
29: ldc           #8
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test21
Start line: 287
End line: 299
Method source code: 
    public void test21() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test21");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "", "UTF-8", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.IllegalArgumentException; message: Must set charset arg to character set of file to parse. Set to null to attempt to detect from HTML");
        } catch (java.lang.IllegalArgumentException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #42
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #8
31: ldc           #24
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #43
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/IllegalArgumentException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test22
Start line: 302
End line: 313
Method source code: 
    public void test22() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test22");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "UTF-8", "hi!");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #45
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #24
29: ldc           #14
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test23
Start line: 316
End line: 321
Method source code: 
    public void test23() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test23");
        java.lang.String str1 = org.jsoup.helper.DataUtil.getCharsetFromContentType("");
        org.junit.Assert.assertNull(str1);
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #46
19: aastore
20: invokevirtual #7
23: pop
24: ldc           #8
26: invokestatic  #34
29: astore_1
30: aload_1
31: invokestatic  #35
34: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test24
Start line: 324
End line: 336
Method source code: 
    public void test24() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test24");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "hi!", "", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #47
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #14
31: ldc           #8
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test25
Start line: 339
End line: 350
Method source code: 
    public void test25() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test25");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "hi!", "hi!");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #48
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #14
29: ldc           #14
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test26
Start line: 353
End line: 365
Method source code: 
    public void test26() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test26");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "UTF-8", "hi!", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #49
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #24
31: ldc           #14
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test27
Start line: 368
End line: 379
Method source code: 
    public void test27() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test27");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "UTF-8", "");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #50
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #24
29: ldc           #8
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test28
Start line: 382
End line: 393
Method source code: 
    public void test28() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test28");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "hi!", "UTF-8");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #51
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #14
29: ldc           #24
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test29
Start line: 396
End line: 408
Method source code: 
    public void test29() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test29");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "UTF-8", "UTF-8", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #52
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #24
31: ldc           #24
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test30
Start line: 411
End line: 423
Method source code: 
    public void test30() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test30");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "UTF-8", "", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #53
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #24
31: ldc           #8
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test31
Start line: 426
End line: 438
Method source code: 
    public void test31() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test31");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "", "", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.IllegalArgumentException; message: Must set charset arg to character set of file to parse. Set to null to attempt to detect from HTML");
        } catch (java.lang.IllegalArgumentException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #54
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #8
31: ldc           #8
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #43
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/IllegalArgumentException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test32
Start line: 441
End line: 452
Method source code: 
    public void test32() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test32");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "", "");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #55
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #8
29: ldc           #8
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test33
Start line: 455
End line: 467
Method source code: 
    public void test33() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test33");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "", "UTF-8", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #56
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #8
31: ldc           #24
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test34
Start line: 470
End line: 481
Method source code: 
    public void test34() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test34");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "UTF-8", "");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #57
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #24
29: ldc           #8
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test35
Start line: 484
End line: 495
Method source code: 
    public void test35() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test35");
        java.io.InputStream inputStream0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(inputStream0, "hi!", "UTF-8");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #58
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #14
29: ldc           #24
31: invokestatic  #15
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test36
Start line: 498
End line: 510
Method source code: 
    public void test36() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test36");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "UTF-8", "hi!", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #59
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #24
31: ldc           #14
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test37
Start line: 513
End line: 525
Method source code: 
    public void test37() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test37");
        java.io.InputStream inputStream0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.load(inputStream0, "UTF-8", "UTF-8", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #60
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #24
31: ldc           #24
33: aload_2
34: invokestatic  #9
37: astore_3
38: ldc           #10
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test38
Start line: 528
End line: 540
Method source code: 
    public void test38() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test38");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "hi!", "UTF-8", parser3);
            org.junit.Assert.fail("Expected exception of type java.nio.charset.IllegalCharsetNameException; message: hi!");
        } catch (java.nio.charset.IllegalCharsetNameException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #61
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #14
31: ldc           #24
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #20
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/nio/charset/IllegalCharsetNameException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test39
Start line: 543
End line: 554
Method source code: 
    public void test39() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test39");
        java.io.File file0 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document3 = org.jsoup.helper.DataUtil.load(file0, "UTF-8", "UTF-8");
            org.junit.Assert.fail("Expected exception of type java.lang.NullPointerException; message: null");
        } catch (java.lang.NullPointerException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #62
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aload_1
27: ldc           #24
29: ldc           #24
31: invokestatic  #17
34: astore_2
35: ldc           #10
37: invokestatic  #11
40: goto          44
43: astore_2
44: return
Exception table:
from    to  target type
26    40    43   Class java/lang/NullPointerException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/RegressionTest0.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/RegressionTest0.class
Class name: RegressionTest0
Method name: test40
Start line: 557
End line: 569
Method source code: 
    public void test40() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest0.test40");
        java.nio.ByteBuffer byteBuffer0 = null;
        org.jsoup.parser.Parser parser3 = null;
        // The following exception was thrown during execution in test generation
        try {
            org.jsoup.nodes.Document document4 = org.jsoup.helper.DataUtil.parseByteData(byteBuffer0, "", "hi!", parser3);
            org.junit.Assert.fail("Expected exception of type java.lang.IllegalArgumentException; message: Must set charset arg to character set of file to parse. Set to null to attempt to detect from HTML");
        } catch (java.lang.IllegalArgumentException e) {
        // Expected exception.
        }
    }
Method byte code: 
0: getstatic     #2
3: ifeq          24
6: getstatic     #3
9: ldc           #4
11: iconst_1
12: anewarray     #5
15: dup
16: iconst_0
17: ldc           #63
19: aastore
20: invokevirtual #7
23: pop
24: aconst_null
25: astore_1
26: aconst_null
27: astore_2
28: aload_1
29: ldc           #8
31: ldc           #14
33: aload_2
34: invokestatic  #19
37: astore_3
38: ldc           #43
40: invokestatic  #11
43: goto          47
46: astore_3
47: return
Exception table:
from    to  target type
28    43    46   Class java/lang/IllegalArgumentException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtilTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtilTest.class
Class name: DataUtilTest
Method name: testCharset
Start line: 13
End line: 20
Method source code: 
    public void testCharset() {
        assertEquals("utf-8", DataUtil.getCharsetFromContentType("text/html;charset=utf-8 "));
        assertEquals("UTF-8", DataUtil.getCharsetFromContentType("text/html; charset=UTF-8"));
        assertEquals("ISO-8859-1", DataUtil.getCharsetFromContentType("text/html; charset=ISO-8859-1"));
        assertEquals(null, DataUtil.getCharsetFromContentType("text/html"));
        assertEquals(null, DataUtil.getCharsetFromContentType(null));
        assertEquals(null, DataUtil.getCharsetFromContentType("text/html;charset=Unknown"));
    }
Method byte code: 
0: ldc           #2
2: ldc           #3
4: invokestatic  #4
7: invokestatic  #5
10: ldc           #6
12: ldc           #7
14: invokestatic  #4
17: invokestatic  #5
20: ldc           #8
22: ldc           #9
24: invokestatic  #4
27: invokestatic  #5
30: aconst_null
31: ldc           #10
33: invokestatic  #4
36: invokestatic  #5
39: aconst_null
40: aconst_null
41: invokestatic  #4
44: invokestatic  #5
47: aconst_null
48: ldc           #11
50: invokestatic  #4
53: invokestatic  #5
56: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtilTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtilTest.class
Class name: DataUtilTest
Method name: testQuotedCharset
Start line: 22
End line: 27
Method source code: 
    @Test public void testQuotedCharset() {
        assertEquals("utf-8", DataUtil.getCharsetFromContentType("text/html; charset=\"utf-8\""));
        assertEquals("UTF-8", DataUtil.getCharsetFromContentType("text/html;charset=\"UTF-8\""));
        assertEquals("ISO-8859-1", DataUtil.getCharsetFromContentType("text/html; charset=\"ISO-8859-1\""));
        assertEquals(null, DataUtil.getCharsetFromContentType("text/html; charset=\"Unsupported\""));
    }
Method byte code: 
0: ldc           #2
2: ldc           #12
4: invokestatic  #4
7: invokestatic  #5
10: ldc           #6
12: ldc           #13
14: invokestatic  #4
17: invokestatic  #5
20: ldc           #8
22: ldc           #14
24: invokestatic  #4
27: invokestatic  #5
30: aconst_null
31: ldc           #15
33: invokestatic  #4
36: invokestatic  #5
39: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtilTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtilTest.class
Class name: DataUtilTest
Method name: discardsSpuriousByteOrderMark
Start line: 29
End line: 34
Method source code: 
    @Test public void discardsSpuriousByteOrderMark() {
        String html = "\uFEFF<html><head><title>One</title></head><body>Two</body></html>";
        ByteBuffer buffer = Charset.forName("UTF-8").encode(html);
        Document doc = DataUtil.parseByteData(buffer, "UTF-8", "http://foo.com/", Parser.htmlParser());
        assertEquals("One", doc.head().text());
    }
Method byte code: 
0: ldc           #16
2: astore_1
3: ldc           #6
5: invokestatic  #17
8: aload_1
9: invokevirtual #18
12: astore_2
13: aload_2
14: ldc           #6
16: ldc           #19
18: invokestatic  #20
21: invokestatic  #21
24: astore_3
25: ldc           #22
27: aload_3
28: invokevirtual #23
31: invokevirtual #24
34: invokestatic  #5
37: return
}
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test0
Start line: 30
End line: 33
Method source code: 
  public void test0()  throws Throwable  {
      PipedInputStream pipedInputStream0 = new PipedInputStream();
      try { 
        DataUtil.load((InputStream) pipedInputStream0, "o,2kF=[k}*vX$jlS", "o,2kF=[k}*vX$jlS");
Method byte code: 
0: new           #2
3: dup
4: invokespecial #3
7: astore_1
8: aload_1
9: ldc           #4
11: ldc           #4
13: invokestatic  #5
16: pop
17: ldc           #6
19: invokestatic  #7
22: goto          32
25: astore_2
26: ldc           #9
28: aload_2
29: invokestatic  #10
32: return
Exception table:
from    to  target type
8    22    25   Class java/io/IOException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test1
Start line: 45
End line: 60
Method source code: 
  public void test1()  throws Throwable  {
      File file0 = MockFile.createTempFile("qPQ", "qPQ");
      Parser parser0 = Parser.htmlParser();
      MockFileInputStream mockFileInputStream0 = new MockFileInputStream(file0);
      // Undeclared exception!
      try { 
        DataUtil.load((InputStream) mockFileInputStream0, (String) null, (String) null, parser0);
        fail("Expecting exception: IllegalArgumentException");
      
      } catch(IllegalArgumentException e) {
         //
         // BaseURI must not be null
         //
         verifyException("org.jsoup.helper.Validate", e);
      }
  }
Method byte code: 
0: ldc           #11
2: ldc           #11
4: invokestatic  #12
7: astore_1
8: invokestatic  #13
11: astore_2
12: new           #14
15: dup
16: aload_1
17: invokespecial #15
20: astore_3
21: aload_3
22: aconst_null
23: checkcast     #16
26: aconst_null
27: checkcast     #16
30: aload_2
31: invokestatic  #17
34: pop
35: ldc           #18
37: invokestatic  #7
40: goto          52
43: astore        4
45: ldc           #20
47: aload         4
49: invokestatic  #10
52: return
Exception table:
from    to  target type
21    40    43   Class java/lang/IllegalArgumentException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test2
Start line: 63
End line: 67
Method source code: 
  public void test2()  throws Throwable  {
      File file0 = MockFile.createTempFile("5&EpZ8M[]AR|~$E[e6 ", "5&EpZ8M[]AR|~$E[e6 ");
      Document document0 = DataUtil.load(file0, (String) null, "5&EpZ8M[]AR|~$E[e6 ");
      assertEquals("5&EpZ8M[]AR|~$E[e6", document0.baseUri());
  }
Method byte code: 
0: ldc           #21
2: ldc           #21
4: invokestatic  #12
7: astore_1
8: aload_1
9: aconst_null
10: checkcast     #16
13: ldc           #21
15: invokestatic  #22
18: astore_2
19: ldc           #23
21: aload_2
22: invokevirtual #24
25: invokestatic  #25
28: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test3
Start line: 70
End line: 82
Method source code: 
  public void test3()  throws Throwable  {
      MockFile mockFile0 = new MockFile("http-equiv");
      try { 
        DataUtil.load((File) mockFile0, "http-equiv", "http-equiv");
        fail("Expecting exception: FileNotFoundException");
      
      } catch(FileNotFoundException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.evosuite.runtime.mock.java.io.MockFileInputStream", e);
      }
  }
Method byte code: 
0: new           #26
3: dup
4: ldc           #27
6: invokespecial #28
9: astore_1
10: aload_1
11: ldc           #27
13: ldc           #27
15: invokestatic  #22
18: pop
19: ldc           #29
21: invokestatic  #7
24: goto          34
27: astore_2
28: ldc           #31
30: aload_2
31: invokestatic  #10
34: return
Exception table:
from    to  target type
10    24    27   Class java/io/FileNotFoundException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test4
Start line: 85
End line: 98
Method source code: 
  public void test4()  throws Throwable  {
      File file0 = MockFile.createTempFile("5&EpZ8M[]AR|~$E[e6 ", (String) null);
      // Undeclared exception!
      try { 
        DataUtil.load(file0, (String) null, (String) null);
        fail("Expecting exception: IllegalArgumentException");
      
      } catch(IllegalArgumentException e) {
         //
         // BaseURI must not be null
         //
         verifyException("org.jsoup.helper.Validate", e);
      }
  }
Method byte code: 
0: ldc           #21
2: aconst_null
3: checkcast     #16
6: invokestatic  #12
9: astore_1
10: aload_1
11: aconst_null
12: checkcast     #16
15: aconst_null
16: checkcast     #16
19: invokestatic  #22
22: pop
23: ldc           #18
25: invokestatic  #7
28: goto          38
31: astore_2
32: ldc           #20
34: aload_2
35: invokestatic  #10
38: return
Exception table:
from    to  target type
10    28    31   Class java/lang/IllegalArgumentException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test5
Start line: 101
End line: 106
Method source code: 
  public void test5()  throws Throwable  {
      Parser parser0 = Parser.htmlParser();
      ByteBuffer byteBuffer0 = ByteBuffer.allocate(14);
      Document document0 = DataUtil.parseByteData(byteBuffer0, "UTF-8", "UTF-8", parser0);
      assertEquals(Document.QuirksMode.noQuirks, document0.quirksMode());
  }
Method byte code: 
0: invokestatic  #13
3: astore_1
4: bipush        14
6: invokestatic  #32
9: astore_2
10: aload_2
11: ldc           #33
13: ldc           #33
15: aload_1
16: invokestatic  #34
19: astore_3
20: getstatic     #35
23: aload_3
24: invokevirtual #36
27: invokestatic  #25
30: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test6
Start line: 109
End line: 124
Method source code: 
  public void test6()  throws Throwable  {
      EvoSuiteFile evoSuiteFile0 = new EvoSuiteFile("http-equiv");
      FileSystemHandling.appendStringToFile(evoSuiteFile0, "http-equiv");
      MockFile mockFile0 = new MockFile("http-equiv");
      // Undeclared exception!
      try { 
        DataUtil.load((File) mockFile0, "http-equiv", "http-equiv");
        fail("Expecting exception: UnsupportedCharsetException");
      
      } catch(UnsupportedCharsetException e) {
         //
         // http-equiv
         //
         verifyException("java.nio.charset.Charset", e);
      }
  }
Method byte code: 
0: new           #37
3: dup
4: ldc           #27
6: invokespecial #38
9: astore_1
10: aload_1
11: ldc           #27
13: invokestatic  #39
16: pop
17: new           #26
20: dup
21: ldc           #27
23: invokespecial #28
26: astore_2
27: aload_2
28: ldc           #27
30: ldc           #27
32: invokestatic  #22
35: pop
36: ldc           #40
38: invokestatic  #7
41: goto          51
44: astore_3
45: ldc           #42
47: aload_3
48: invokestatic  #10
51: return
Exception table:
from    to  target type
27    41    44   Class java/nio/charset/UnsupportedCharsetException
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test7
Start line: 127
End line: 130
Method source code: 
  public void test7()  throws Throwable  {
      String string0 = DataUtil.getCharsetFromContentType("5&EpZ8M[]AR|~$E[e6 ");
      assertNull(string0);
  }
Method byte code: 
0: ldc           #21
2: invokestatic  #43
5: astore_1
6: aload_1
7: invokestatic  #44
10: return
Source File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/src/test/java/org/jsoup/helper/DataUtil_ESTest.java
Bytecode File: /data/islam/DBT-workbench/resources/subjects/fixed/jsoup/27/target/test-classes/org/jsoup/helper/DataUtil_ESTest.class
Class name: DataUtil_ESTest
Method name: test8
Start line: 133
End line: 136
Method source code: 
  public void test8()  throws Throwable  {
      String string0 = DataUtil.getCharsetFromContentType((String) null);
      assertNull(string0);
  }
Method byte code: 
0: aconst_null
1: checkcast     #16
4: invokestatic  #43
7: astore_1
8: aload_1
9: invokestatic  #44
12: return
}
